store-implementation/parquet-store/src/test/java/uk/gov/gchq/gaffer/parquetstore/utils/ConvertViewToFilterTest.java

```diff
@@ -39,13 +45,14 @@ public void setUp() throws StoreException {
                 getClass().getResourceAsStream("/schemaUsingTypeValueVertexType/dataTypes.json"),
```

Comment:
You could load the Schema like this:

schema = Schema.fromJson(StreamUtil.schemas(ConvertViewToFilterTest.class, "schemaUsingTypeValueVertexType"));